rig/tests/par/test_wrapper.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+from six import iteritems
+
+from rig.machine import Machine, Cores, SDRAM
+from rig.netlist import Net
+
+from rig.par import par
+
+
+class TestWrapper(object):
+    """Simple santy-check level tests of the wrapper, no comprehensive checks
+    since internal function is largely tested elsewhere."""
+
+    def test_empty(self):
+        # Simplest possible case of an empty system
+        m = Machine(1, 1)
+        placements, allocations, application_map, routing_tables = \
+            par({}, {}, [], {}, m)
+        assert placements == {}
+        assert allocations == {}
+        assert application_map == {}
+        assert routing_tables == {}
+
+    def test_ring(self):
```

Comment:
Highlight for @mundya: This test contains a full run-through of the place/route process for a simple graph.